shed/event_streams.py

```diff
@@ -189,7 +192,7 @@ def emit(self, x):
 
         """
         if x is not None:
-            self.curate_streams(x, True)
+            x = self.curate_streams(x, True)
```

Comment:
I believe so. I found this on the actual pipeline implementation side of xpdAn and so I made a test which failed on this side and then fixed the much simpler test, so it should be covered now.